README.md

```diff
@@ -15,6 +15,7 @@ Documentation for the [mLRS project](https://github.com/olliw42/mLRS).
     - [CRSF Sensors](docs/CRSF_SENSORS.md)
 - Setup for [MAVLink for OpenTx](docs/MAVLINK_FOR_OPENTX.md)
 - Basic setup for [SBus Radios](docs/BASIC_SETUP.md)
+- Setup for [ArduPilot](docs/ARDUPILOT.md)
```

Comment:
I would not use the word "Setup" here, as the content provided here is different in nature to the other "Setup"s. I don't have an immediate good suggestion, but maybe something like "Configuration Options for ArduPilot Systems".

Reply:
- Rephrased to 'Configuration Options for ArduPilot Systems'
- Page itself is just 'ArduPilot Systems'